### PR TITLE
removed DotNetNuke.Web.Deprecated

### LINKED ISSUE #28: Telerik Dependency
## Please summarize your question in one sentence
Are there plans to remove dependency on Telerik?

## Give a more extended description
Mitchell Sellers DNNTelerikIdentifier tool shows that this module (Dnn.Modules.Newsletters.dll) has a dependency on Telerik. Are there plans to remove that dependency?

### PR DESCRIPTION
**Pull request by gncube**
## Changes made
- Telerik removed 

## PR Template Checklist
- [x] Telerik removal 

## Please mark which issue is solved

Close #28

### CODE REVIEW

File: Dnn.Modules.Newsletters.csproj [hunk 1]
```diff
@@ -101,22 +96,17 @@
...
-      <SpecificVersion>False</SpecificVersion>
-      <Private>False</Private>
-    </Reference>
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Components\NewsletterController.cs" />
-    <Compile Include="Newsletter.ascx.designer.cs">
-      <DependentUpon>Newsletter.ascx</DependentUpon>
-    </Compile>
-    <Compile Include="Properties\AssemblyInfo.cs" />
```
Comment (valadas) [inline Dnn.Modules.Newsletters.csproj:115]: Was this intentaional ?
the rest looks right to me.

Comment (david-poindexter) [reply Dnn.Modules.Newsletters.csproj:115]: @valadas these lines are still in the file (see https://github.com/DNNCommunity/DNN.Newsletter/pull/30/files#diff-6cc25bcd8a5814ee68133c156f31b2878515fa71bfbfcc32142242cd80a5bffeR106-R109) so I think this is fine.

File: dnn_Newsletters.dnn [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<dotnetnuke type="Package" version="5.0">
   <packages>
-    <package name="DotNetNuke.Newsletters" type="Module" version="08.01.00">
+    <package name="DotNetNuke.Newsletters" type="Module" version="08.01.01">
```
Comment (david-poindexter) [inline dnn_Newsletters.dnn:3]:
```suggestion
    <package name="DotNetNuke.Newsletters" type="Module" version="08.02.00">
```

I believe this warrants a minor release.

**Review by david-poindexter — CHANGES_REQUESTED** [review]
This looks great @gncube - thanks so much for the contribution! 

With incrementing the version, I believe we also need to increment in the following places as well:
* https://github.com/DNNCommunity/DNN.Newsletter/blob/master/Properties/AssemblyInfo.cs#L36-L37
* https://github.com/DNNCommunity/DNN.Newsletter/blob/d5a6685be68e44b307d11c5bad15f6ddbe39d360/releaseNotes.txt

File: Dnn.Modules.Newsletters.csproj (hunk 1 above)
Comment (valadas) [reply Dnn.Modules.Newsletters.csproj:115]: Oh, sorry, did not notice this 😄

**Review by aspelt — APPROVED** [review]
Good work!

**Pull request closed without merging**